tests/UI/campaigns/productV2/sanity/01_filterProducts.js

```diff
@@ -0,0 +1,168 @@
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+
+// Import utils
+const helper = require('@utils/helpers');
+const testContext = require('@utils/testContext');
+
+// Import login steps
+const loginCommon = require('@commonTests/BO/loginBO');
+const {enableNewProductPageTest, disableNewProductPageTest} = require('@commonTests/BO/advancedParameters/newFeatures');
+
+// Import pages
+const dashboardPage = require('@pages/BO/dashboard');
+const productsPage = require('@pages/BO/catalog/productsV2');
+
+// Import data
+const {Products} = require('@data/demo/products');
+const {Categories} = require('@data/demo/categories');
+
+const baseContext = 'productV2_sanity_filterProducts';
+
+let browserContext;
+let page;
+let numberOfProducts = 0;
+
+describe('BO - Catalog - Products : Filter in Products Page', async () => {
+  // Pre-condition: Enable new product page
+  enableNewProductPageTest(`${baseContext}_enableNewProduct`);
+
+  // before and after functions
+  before(async function () {
+    browserContext = await helper.createBrowserContext(this.browser);
+    page = await helper.newTab(browserContext);
+  });
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+  });
+
+  describe('Filter products table by : ID, Name, Reference, Category, Price, Quantity and Status', async () => {
+    it('should login in BO', async function () {
+      await loginCommon.loginBO(this, page);
+    });
+
+    it('should go to \'Catalog > Products\' page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToProductsPage', baseContext);
+
+      await dashboardPage.goToSubMenu(
+        page,
+        dashboardPage.catalogParentLink,
+        dashboardPage.productsLink,
+      );
+
+      const pageTitle = await productsPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(productsPage.pageTitle);
+    });
+
+    it('should check that no filter is applied by default', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkNoFilter', baseContext);
+
+      const isVisible = await productsPage.isSearchButtonVisible(page);
```

Comment:
Only this part surpises me, because the search button is always visible But it's disabled until a filter is typed so shouldn't it test that the button is disabled instead considering the test here is supposed to be `'should check that no filter is applied by default'`

Reply:
Thank you @jolelievre for your review :)
Yes nice catch ! I should test that reset button is not visible and not search button
I will fix it :)

Reply:
Ah yes indeed, it seems more useful